.github/workflows/static_code_checks.yaml

```diff
@@ -44,8 +44,22 @@ jobs:
           # ignoring security vulnerabilities in cryptography
           # because flwr 1.7 depends on them. Opening an issue
           # with flwr. Likely can remove this in flwr 1.8.
+          #
+          # ignoring security vulnerabilities for mlflow as monai's
+          # autoseg3d pipeline depends on mlflow=^2.12.2
+          # Raised an issue with mlflow
+          # The 1st 4 vulnerabilities are for cryptography, the rest (9) are for mlflow
```

Comment:
I have fixed the cryptography ones in a PR that I just put up so no need to mention them. But I do like how you have indicated in a comment that the vulns are from mlflow because going back after a while its hard to remember at first glance. Maybe you can say GHSA-x38x-g6gr-jqff to GHSA-7p8j-qv6x-f4g4 originate from mlflow